specification/implementation/structure/folders/index.md

```diff
@@ -54,4 +54,4 @@ The implementation requirements of the CSIP Information Package structure are:
 **CSIPSTR15**: We recommend including all XML schema documents for any structured metadata within package. These schema documents **SHOULD** be placed in a sub-folder called `schemas` within the Information Package root folder and/or the representation folder.
 
 <a name="CSIPSTR16"></a>
-**CSIPSTR16**: We recommend including any supplementary documentation for the package or a specific representation within the package. Supplementary documentation **SHOULD** be placed in a sub-folder called `documentation` within the Information Package root folder and/or the representation folder.
+**CSIPSTR16**: We recommend including any supplementary documentation for the package or a specific representation within the package. Supplementary documentation **SHOULD** be placed in a sub-folder called `documentation` within the Information Package root folder and/or the representation folder. Examples of documentation includes representation information and manuals for the system the data objects have been exported from. 
```

Comment:
We recommend including any supplementary documentation for the package or a specific representation within the package. Supplementary documentation **SHOULD** be placed in a sub-folder called `documentation` within the Information Package root folder and/or the representation folder. Examples of documentation include representation information and manuals for the system the data objects were exported from.

Reply:
Updated made.